config/routes_with_auth.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  devise_for :users
+
+  use_doorkeeper scope: 'api/v1/sessions' do
+    skip_controllers :authorizations, :applications, :authorized_applications
+    controllers tokens: 'api/v1/oauth_tokens'
```

Comment:
You are defining here a route for oauth_tokens but the controller is not defined